src/loader.ts

```diff
@@ -27,6 +27,10 @@ class Loader extends LitElement {
   @property({ type: String }) extraMsg?: string;
   @property({ type: String }) swName?: string;
 
+  private get isLoadingWacz() {
+    return this.sourceUrl?.toLowerCase().endsWith(".wacz");
```

Comment:
@ikreymer Is there a better way to determine whether the currently loading file is WACZ?

Reply:
There is detection in the service worker, but that requires peeking into the file..
Perhaps we show the spinner, until/unless we get a progress that's not 0, and then switch to progress bar?